pkg/cli/bootstrap.go

```diff
@@ -72,6 +72,15 @@ func overrideConfigLoaders(honeycombAPIKey, dataset string, tracingDebug bool) {
 					SamplePercent: 100,
 				},
 			}
+			traceConfig.Otel = trace.Otel{
+				Enabled:       traceConfig.Honeycomb.Enabled,
+				Endpoint:      traceConfig.Honeycomb.APIHost,
```

Comment:
You may need to manually set this one to `api.honeycomb.io`. Otel will fail to send traces if you provide the scheme.